src/connection_string.ts

```diff
@@ -326,13 +326,20 @@ export function parseOptions(
   for (const key of allKeys) {
     const values = [];
     if (objectOptions.has(key)) {
-      values.push(objectOptions.get(key));
+      const options = Array.isArray(objectOptions.get(key))
```

Comment:
nit: this could be simplified by doing `[].concat(objectOptions.get(key))` in these 3 places. It looks a little odd but does the same without accessing the options twice or the need for the ternary. Could also make it a util function called `wrapInArray` or something similar.